src/components/Home.tsx

```diff
@@ -17,11 +21,23 @@ const Title = styled.h1<HeadingProps>`
   ${typography}
 `;
 
+const LangButton = styled.button<HeadingProps>`
+  ${color}
+  ${space}
+  ${typography}
+  ${border}
+`;
+
 const Home = () => {
+  const { t } = useTranslation();
   return (
-    <Title color="primary" bg="error" p={3} fontSize={3}>
-      Oxymore
-    </Title>
+    <main>
+      <Title color="primary" bg="error" p={3} fontSize={3}>
+        {t("home.header")}
+      </Title>
+      <LangButton onClick={() => i18next.changeLanguage("en")}>EN</LangButton>
```

Comment:
instead of adding the onClick handlers inline here can you extract them out and use `useCallback`
https://reactjs.org/docs/hooks-reference.html#usecallback

it will end up looking more like:
```
  const onLangClick = useCallback(
    (countryId: string) => () => i18next.changeLanguage(countryId),
    []
  );

...
      <LangButton onClick={onLangClick("en")}>EN</LangButton>

```

Reply:
the reason for this is that it will stop the component from rerendering everytime

Reply:
thanks Justen, had a compile error due to TypeScript when using `useCallback` but did a quick fix and now working fine.